lib/rspec/core/notifications.rb

```diff
@@ -199,6 +199,12 @@ def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::Console
         @exception_presenter.fully_formatted(failure_number, colorizer)
       end
 
+      # @return [Array] The failure information fully formatted in the way that
+      #   RSpec's built-in formatters emit, split by line.
+      def fully_formatted_lines(failure_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes)
+        @exception_presenter.fully_formatted_lines(failure_number, colorizer)
+      end
```

Comment:
Is it really worth expanding our public API for this?  After all, isn't calling this the same as calling `notification.fully_formatted(...).lines`?  If so, I'd rather not widen our API (and thus, increase our maintenance burden) when it's so simple to get all the lines already.